colossalai/inference/core/engine.py

```diff
@@ -149,11 +149,17 @@ def init_model(self, model_or_path: Union[nn.Module, str], model_policy: Policy
             )
 
         if model_policy is None:
-            if self.inference_config.pad_input:
-                model_type = "padding_" + self.model_config.model_type
-            else:
-                model_type = "nopadding_" + self.model_config.model_type
-            model_policy = model_policy_map[model_type]()
+            prefix = "nopadding" if not self.inference_config.pad_input else "padding"
```

Comment:
Since there is currently no padding code in our framework, do we need to delete this section of code?

Reply:
I've considered this issue, while I think `pad_input` could be reserved for now. And this PR has fixed the error message and it will report the lack of padding policy if anyone turns it on